twindle-cli/twitter/utils/process-tweet-entities.js

```diff
@@ -77,6 +77,9 @@ function renderMedia(tweetObj) {
  */
 function renderOutsiderLinks(tweetObj) {
   /** @type {any[]} */
+  
+  if(!tweetObj.entities) return tweetObj;
```

Comment:
This might fail, if `tweetObj` is null or undefined in rare case scenarios.

Reply:
tweetObj is our parameter so that failing will most likely not happen. But if your have a better way of tackling this issue let me know. I can change my file accordingly

Reply:
Good suggestion @arthurbuhl 
Optional chaining will be helpful for these cases, but that's for later
Let's keep fixing issues as they appear.